includes/api/class-wc-rest-products-controller.php

```diff
@@ -57,8 +87,808 @@ public function register_routes() {
+					'enum'        => array( 'taxable', 'shipping', 'none' ),
+					'context'     => array( 'view', 'edit' ),
+				),
+				'tax_class' => array(
+					'description' => __( 'Tax class.', 'woocommerce' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'managing_stock' => array(
+					'description' => __( 'Stock management at product level.', 'woocommerce' ),
+					'type'        => 'boolean',
+					'default'     => false,
+					'context'     => array( 'view', 'edit' ),
+				),
+				'stock_quantity' => array(
+					'description' => __( 'Stock quantity.', 'woocommerce' ),
+					'type'        => 'integer',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'in_stock' => array(
+					'description' => __( 'Controls whether or not the product is listed as "in stock" or "out of stock" on the frontend.', 'woocommerce' ),
+					'type'        => 'boolean',
+					'default'     => true,
+					'context'     => array( 'view', 'edit' ),
+				),
+				'backorders' => array(
+					'description' => __( 'If managing stock, this controls if backorders are allowed.', 'woocommerce' ),
+					'type'        => 'string',
+					'default'     => 'no',
+					'enum'        => array( 'no', 'notify', 'yes' ),
+					'context'     => array( 'view', 'edit' ),
+				),
+				'backorders_allowed' => array(
+					'description' => __( 'Shows if backorders are allowed.', 'woocommerce' ),
+					'type'        => 'boolean',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'backordered' => array(
+					'description' => __( 'Shows if a product is on backorder.', 'woocommerce' ),
+					'type'        => 'boolean',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'sold_individually' => array(
+					'description' => __( 'Allow one item to be bought in a single order.', 'woocommerce' ),
+					'type'        => 'boolean',
+					'default'     => false,
+					'context'     => array( 'view', 'edit' ),
+				),
+				'weight' => array(
+					'description' => sprintf( __( 'Product weight (%s).', 'woocommerce' ), get_option( 'woocommerce_weight_unit' ) ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'length' => array(
+					'description' => sprintf( __( 'Product length (%s).', 'woocommerce' ), get_option( 'woocommerce_dimension_unit' ) ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'width' => array(
+					'description' => sprintf( __( 'Product width (%s).', 'woocommerce' ), get_option( 'woocommerce_dimension_unit' ) ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'height' => array(
+					'description' => sprintf( __( 'Product height (%s).', 'woocommerce' ), get_option( 'woocommerce_dimension_unit' ) ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'shipping_required' => array(
```

Comment:
Schema is for all data and we should return all data too for all types of products.